GTG/core/versioning.py

```diff
@@ -25,7 +25,7 @@
 from lxml import etree as et
 from GTG.core.dates import Date
 from GTG.core.dirs import DATA_DIR
-from GTG.core import xml
+#from GTG.core import xml
```

Comment:
What is this? Can be just delete the line?

Reply:
I've deleted it now. Thanks!